src/transforms/transform_thumbnail.stories.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { transform_thumbnail } from "./column_transforms";
+import { TransformExample } from "./utils/TransformExample";
+
+<Meta title="Utils/Transforms/Thumbnail"/>
+
+# transform_thumbnail
+
+Convert a Date object into a month / day / year string.
```

Comment:
Just a spelling suggestion, conjugate the convert verb in the third person, also this description corresponds to the `transform_dateFormat` change it to something like:

**Converts** a thumbnail from a given height and width.

Reply:
Nice catch, didn't notice it